etl/steps/data/garden/energy/2024-05-08/global_primary_energy.meta.yml

```diff
@@ -234,6 +234,8 @@ tables:
         description_short: *description_short_pct_direct
         display:
           name: Modern biofuels
+        presentation:
```

Comment:
I add it here because I can't comment line 12: I think it should be `combines` instead of `combined`